### DISCUSSION TITLE
Issues with Docker Proxy Environment variables

### DISCUSSION THREAD

**Discussion by sebocampo** [DependencyTrack/dependency-track#3310] [Q&A]
Hello,
I have deployed this through the bundled Docker installation route on a _local_ Openshift cluster. When importing the BOM, no vulnerabilities are appearing. The case seems to be that the download of the vulnerabilities database seems to be blocked from company networks. I am experiencing an issue with my company proxy using NTLM. I have tried defined the proxy environment variables for both JAVA_TOOL_OPTIONS (proxyUser, proxyPassword, proxyHost, proxyPort, nonProxyHosts, jdk.http.auth.proxying.disabledSchemes, jdk.http.auth.tunneling.disabledSchemes) and the respective Alpine variables (ALPINE_HTTP_PROXY, etc).
I've followed the instructions in both configuration (https://docs.dependencytrack.org/getting-started/configuration/#proxy-configuration) and Docker deployment (https://docs.dependencytrack.org/getting-started/deploy-docker/). 

I've found other tickets and attempted solutions recommended but I keep seeing the same errors listed below. 

```[NistMirrorTask] Initiating download of https://nvd.nist.gov/feeds/json/cve/1.1/nvdcve-1.1-2021.json.gz 
WARN [ProxyAuthenticationStrategy] Authentication scheme Negotiate not supported
WARN [NistMirrorTask] Unable to download - HTTP Response 407: Proxy Authentication Required
ERROR [NistMirrorTask] An error occurred uncompressing NVD payload
```
This repeats for all vulnerabilities.
I'm using the correct format for all my credentials following various documentation sources. 

I'm reaching out to see if there's certain flags I may be missing or if there's a way to get the internal Apache HTTP Client to utilize my system-settings for proxy & authentication. Is there some sort of external control available for these Apache settings?

Thanks,

**Comment by lbianchinisigma** [comment]
Did anyone ever find a solution to this problem? I have the same exact situation and behaviour and i'm struggling to find a solution; i've succesfully setup a docker solution on a VM, but i faced this problem on a kubernetes environment.
